vertical-pod-autoscaler/enhancements/4943-vpa-scope/README.md

```diff
@@ -0,0 +1,965 @@
+-   [ ](R) Design details are appropriately documented
+-   [ ](R) Test plan is in place, giving consideration to SIG Architecture and
+    SIG Testing input (including test refactors)
+    -   [ ] e2e Tests for all Beta API Operations (endpoints)
+    -   [ ](R) Ensure GA e2e tests meet requirements for
+        [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+    -   [ ](R) Minimum Two Week Window for GA e2e tests to prove flake free
+-   [ ](R) Graduation criteria is in place
+    -   [ ](R)
+        [all GA Endpoints](https://github.com/kubernetes/community/pull/1806)
+        must be hit by
+        [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+-   [ ](R) Production readiness review completed
+-   [ ](R) Production readiness review approved
+-   [ ] "Implementation History" section is up-to-date for milestone
+-   [ ] User-facing documentation has been created in [kubernetes/website], for
+    publication to [kubernetes.io]
+-   [ ] Supporting documentation—e.g., additional design documents, links to
+    mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. AEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+This AEP proposes extending the Vertical Pod Autoscaler (VPA) to support use
+cases where each pod under a DaemonSet should have different resource requests
+and limits. This is implemented by adding a `Scope` field to the
+VerticalPodAutoscaler API, which initially supports a value of `node`,
+indicating that the VPA recommendation should be calculated and applied
+independently for pods scheuled to the same node.
```

Comment:
A little late to the party here, I'm a little confused on the wording here. For pods under DaemonSets, why should recommendations be calculated independently for pods schedule to the same node? Is this supposed to say pods scheduled to different nodes (which is already implied when using a DaemonSet)?